test_programs/cl8/3.lox

```diff
@@ -0,0 +1,24 @@
+---
+expected_error_type: none
+---
+class Counter {
+  init(startValue) {
+    if (startValue < 0) {
+      print "startValue can't be negative";
```

Comment:
test 1 & 2 here look okay - they test the constructor behaviour specifically and aren't too complex. test 4 is okay too, tests a combination. test 3 here could instead test calling init on an instance